src/compiler/worker/getConfig.js

```diff
@@ -14,10 +14,15 @@ module.exports = function getConfig(
   configOptions: any,
   platform: Platform
 ) {
-  return makeReactNativeConfig(
+  let config;
+  try {
     // $FlowFixMe
-    require(configPath),
-    configOptions,
-    platform
-  );
+    config = require(configPath);
```

Comment:
Does this work with the current code? The config should actually be `require(configPath).default` in this case since the export is ES export, and we're importing with node's `require`

Reply:
https://github.com/callstack/haul/blob/fde95a923d288b91c3fdeba301275a8692123b7d/src/utils/makeReactNativeConfig.js#L236-L237